src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/styles.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+
+import {Color} from 'app/utils/theme';
+import space from 'app/styles/space';
+
+// TODO(style): the color #fffcfb and  #e7c0bc are not yet in theme and no similar theme's color was found.
+const BreadCrumb = styled('li')<{error?: boolean}>`
```

Comment:
Should we rename to isError so that this prop is not passed to native li element and we avoid having errors related to assigning boolean value to string...?

Reply:
sure! I am going to ship this change in a upcoming PR 😉